message/retrylater.go

```diff
@@ -23,19 +23,17 @@ type retryLaterHandler struct {
 }
 
 func (r *retryLaterHandler) Do(ctx context.Context, orig Handler, message *servicebus.Message) Handler {
```

Comment:
Note: I believe this is a result of a previous PR that made the auto lock renewal, but the following should be updated:

The comment here should probably be updated:
https://github.com/Azure/go-shuttle/blob/bab7408d88474b667938609459f1bdb8cd6e28c3/message/retrylater.go#L14

Also, the README.md for go-shuttle:
https://github.com/Azure/go-shuttle#postponed-handling-of-message

Reply:
Simply should add some info about how the auto lock renewal affects this

Reply:
that assumes that you enable auto lock renewal :). but yeah, we know the behavior here. I'll update the comment